Sources/EpoxyBars/BarInstaller/BarWrapperView.swift

```diff
@@ -191,7 +191,10 @@ public final class BarWrapperView: UIView {
       // The behavior is configured regardless of content equality sice behavior is not equatable.
       model.configureBehavior(view, traitCollection: traitCollection)
     } else {
-      let view = makeView(from: model, animated: animated)
+      let view = makeView(
+        from: model,
+        originalModel: originalModel as? InternalBarModeling,
```

Comment:
Possible `AnyBarModel` type is passed to `makeView` function so `willDisplay` can be called. If we want this to be more targeted to `AnyBarModel`, we may want to try to optionally cast to `AnyBarModel` instead to ensure that we don't double call `willDisplay`

Reply:
@erichoracek , what do you think about trying to cast `originalModel` to `AnyBarModel` instead of `InternalBarModeling`? It's hard to tell if `originalModel` can be a `BarModel` since it's already been type-erased to `BarModeling` in the public API. When I removed my `AnyBarModel` testing wrapper, I didn't see the underlying `willDisplay` closure get called twice, which would happen if `originalModel` is actually `BarModel`. However, there could be another layer of `eraseToAnyBarModel` that is called that I'm not seeing.

Reply:
Looks like `BarStackView` already [type-erases](https://github.com/airbnb/epoxy-ios/blob/a527b22cb7afb84e4b88b030d4a739fda981a64b/Sources/EpoxyBars/BarInstaller/BarStackView.swift#L41) to `AnyBarModel`, so if all paths go through there it might be guaranteed that `originalModel` is always `AnyBarModel`

Reply:
Hmm let's keep it the way it is just because `BarWrapperView` is public